src/azure-cli/azure/cli/command_modules/rdbms/_client_factory.py

```diff
@@ -324,5 +324,4 @@ def resource_client_factory(cli_ctx, **_):
 
 
 def network_client_factory(cli_ctx):
-    from azure.mgmt.network import NetworkManagementClient
-    return get_mgmt_service_client(cli_ctx, NetworkManagementClient, api_version="2018-08-01")
+    return get_mgmt_service_client(cli_ctx, ResourceType.MGMT_NETWORK)
```

Comment:
actually you even don't need to define it here. network module already has the definition and you could just import it.
https://github.com/Azure/azure-cli/blob/0a90d440930c38b2a7fa8ecf6f500cfb55ae1ca4/src/azure-cli/azure/cli/command_modules/network/_client_factory.py#L7-L10